### ISSUE TITLE
Android support?

### ISSUE THREAD

**Issue by RamonGebben** [artemyarulin/react-native-xml#1]
It would be great if this would support android as wel.

**Comment by artemyarulin** [comment]
Yep, unfortunately I don't know anything about Android development, so pull requests are welcome.

Another thing that you could try to use https://github.com/jindw/xmldom, which allows to parse XML using pure JavaScript, so JavaScriptCore should be supported.

Regarding XPath - you can try this library https://www.npmjs.com/package/xpath

I'm not sure if it will work - but in theory it should :)
